src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -510,31 +510,28 @@ public enum WebRTCStatisticsCategory
         RemoteOutboundRtp = 0x0010,
 
         /// <summary>
-        /// All types of WebRTC statistics.
+        /// Candidate pair.
         /// </summary>
-        All = Codec | InboundRtp | OutboundRtp | RemoteInboundRtp | RemoteOutboundRtp
-    }
-
-    [Flags]
-    internal enum WebRTCStatisticsPropertyCategory
-    {
-        Common = 0x00000100,
-
-        Codec = 0x00000200,
+        /// <since_tizen> 11 </since_tizen>
+        CandidatePair = 0x0020,
 
-        RtpStream = 0x00000400,
-
-        ReceivedRtpStream = 0x00000800,
-
-        InboundRtpStream = 0x00001000,
-
-        SentRtpStream = 0x00002000,
-
-        OutboundRtpStream = 0x00004000,
+        /// <summary>
+        /// Local candidate.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        LocalCandidate = 0x0040,
 
-        RemoteInboundRtpStream = 0x00008000,
+        /// <summary>
+        /// Remote candidate.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        RemoteCandidate = 0x0080,
 
-        RemoteOutboundRtpStream = 0x00010000
+        /// <summary>
+        /// All types of WebRTC statistics.
+        /// </summary>
+        [Obsolete("Deprecated since API11; Will be removed in API13.")]
+        All = Codec | InboundRtp | OutboundRtp | RemoteInboundRtp | RemoteOutboundRtp | CandidatePair | LocalCandidate | RemoteCandidate
```

Comment:
changing of existing enum values are not normally allowed, an argument can be made that this holds the purpose of the "All" unchanged

Reply:
I've got approval for change from PM part.